torch/sparse/_triton_ops.py

```diff
@@ -835,7 +844,50 @@ def kernel(grid, *sliced_tensors):
             size=input.shape,
             layout=input.layout
         )
+
+    def _scaled_dot_product_attention(
+        query: torch.Tensor,
+        key: torch.Tensor,
+        value: torch.Tensor,
+        attn_mask: Optional[torch.Tensor],
+        dropout_p: float = 0.0,
+        is_causal: bool = False
```

Comment:
scale argument?

Reply:
It is not in the docs, or am I missing something?

Reply:
https://pytorch.org/docs/main/generated/torch.nn.functional.scaled_dot_product_attention.html#torch.nn.functional.scaled_dot_product_attention

Reply:
Oops, sorry, missed it